demo/molecule/select/demo/index.js

```diff
@@ -124,6 +124,26 @@ const Demo = () => (
       </MoleculeSelect>
     </div>
 
+    <div className={CLASS_DEMO_SECTION}>
+      <h3>With state</h3>
+      <p>
+        State to highlight that can be <code>success</code>, <code>error</code>{' '}
+        or <code>alert</code>
+      </p>
+      <MoleculeSelect
+        placeholder="Select a Country..."
+        onChange={(_, {value}) => console.log(value)}
+        iconArrowDown={<IconArrowDown />}
+        state="alert"
```

Comment:
if you expose `SELECT_STATES`, it would be great to have an example of this implementation instead of an `"alert"` string:

```javascript

state={SELECT_STATES.ALERT}
```

Reply:
True! Thanks!

Reply:
Fixed!